### PR TITLE
Fix #11376: Added No result thumbnail to Home Page

### LINKED ISSUE #11376: [MU4 Issue] Scores - Search - No results causes icon to disappear with out context
**Describe the bug**
In the Scores tab when you search for a score and no results are found, all items disappear (including new score icon) with no context (ex: "Sorry, no results found").

**To Reproduce**
1. Go to 'Scores'
2. Type something in search bar that will not be found
3. Watch window contents disappear

**Expected behavior**
If no results are found, there should be some contextual cues to let the user know that nothing is found, for example text.

**Screenshots**

https://user-images.githubusercontent.com/30370609/165254585-1347104d-35ce-46e2-8c4d-9809aea4279c.mov

**Platform information**
OS: macOS 12.2
Arch.: x86_64
MuseScore version (64-bit): 4.0.0-2224366007
revision: github-musescore-musescore-9a7fabf

### PR DESCRIPTION
**Pull request by HemantAntony**
Resolves: #11376

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by Tantacrul** [comment]
OK, this is perfect... apart from one colour choice of mine that was not thought through properly.

Can you change the BG of this rectangle:
image

To use:
Colour: **BACKGROUND_PRIMARY_COLOUR**
Opacity: **255 / 100%**

(My old colours made it a bit hard to read in dark mode).

No need to create another PR to be checked. It's good to go once you make that little change.

**Comment by Tantacrul** [comment]
Thanks a lot, by the way @HemantAntony! Always great to get these UX issues out of the way. Just adds to the overall finesse of the app!

**Comment by HemantAntony** [comment]
@Tantacrul Happy to help!

I updated the PR again. So this is now ready for review/merge

**Comment by Tantacrul** [comment]
@RomanPudashkin or @Eism - if you're happy with this, can it be sent to merge city?

**Comment by Tantacrul** [comment]
Oh, I see @Eism has already been assigned as a reviewer. In that case, he will get around to it when he has time. I didn't realise I had already tested it!

File: src/project/qml/MuseScore/Project/internal/ScoreItem.qml [hunk 1]
```diff
@@ -209,6 +214,20 @@ FocusScope {
...
+    Component {
+        id: noResultFoundComp
+
+        Rectangle {
+            anchors.fill: parent
+            color: ui.theme.backgroundPrimaryColor
+
+            StyledTextLabel {
+                anchors.fill: parent
+                text: "No results found"
```
Comment (Eism) [inline src/project/qml/MuseScore/Project/internal/ScoreItem.qml:226]: Please add a translation

**Comment by HemantAntony** [comment]
@Eism Ready for merge if no other problems :)

**Review by Eism — APPROVED** [review]

**Pull request merged**